src/context.ts

```diff
@@ -25,19 +25,19 @@ export class Context {
     public readonly viteConfig: ResolvedConfig,
   ) {
     this.options = resolveOptions(options, viteConfig)
-    const { globs, dirs } = this.options
+    const { watchGlobs, dirs } = this.options
```

Comment:
Can we just use the `globs` here? Maybe we just need to normalization to replace black slash to slash?

Reply:
Just saw your comment, I will merge this for now and seeking for a better solution when I get access to a Windows PC. Thank you!

Reply:
Yeah, we could do the backslash replacement too.  It's kind of a choice between two messy things.  If you want me to test any other alternatives let me know; happy to help

Reply:
I think it has a lighter mental burden to "always works on slash". My try on #23 works for Mac, if you don't mind, I'd be grateful if you'd like to help on testing out. (guess it's time for me to setup e2e test then, always been too lazy to do so 😅)